itou/utils/admin.py

```diff
@@ -33,3 +36,46 @@ def formfield_for_dbfield(self, db_field, request, **kwargs):
         if isinstance(field, gis_fields.PointField):
             field.widget = OSMWidget(attrs={"map_width": 800, "map_height": 500, "CSP_NONCE": request.csp_nonce})
         return field
+
+
+class ItouTabularInline(TabularInline):
+    list_select_related = []
+
+    def get_queryset(self, request):
+        select_related_fields = set()
+        limit_to = set(self.list_select_related) | set(self.get_fields(request))
```

Comment:
Je trouve cela bizarre de rechecker que `self.list_select_related` soit bien éligible à être un select_related.
Si on se retrouve dans `if not field.is_relation or field.auto_created or field.many_to_many:` je lèverais bien une erreur indiquant `list_select_related` doit être corrigé.